### ISSUE TITLE
OpenAPI is invalid

### ISSUE THREAD

**Issue by NadChel** [cezarykluczynski/stapi#49]
Hey! I checked [your OpenAPI](https://stapi.co/api/v1/rest/common/download/stapi.yaml), and it seems to be incorrect. I tried a few endpoints, and in all cases I got a 500. For example, I hit:

* https://stapi.co/api/v1/rest/animal?uid=1
* https://stapi.co/api/v1/rest/astronomicalObject?uid=1
* https://stapi.co/api/v2/rest/astronomicalObject?uid=1
* https://stapi.co/api/v1/rest/book?uid=1
 
I realize it's a volunteer project so I don't have any complaints, but I figured you should be informed 🙂

**Comment by kiwdave** [comment]
where did you obtain the uid? fir example: https://stapi.co/api/v1/rest/animal?uid=ANMA0000028270

**Comment by NadChel** [comment]
> where did you obtain the uid? for example: https://stapi.co/api/v1/rest/animal?uid=ANMA0000028270

I don't believe 500 is an appropriate status code if no record with a given id (e.g. 1) exists

**Comment by kiwdave** [comment]
gotcha, yeah 404 would be more appropriate.

**Comment by NadChel** [comment]
Don't worry, I edited the title just a few hours back. It used to be more generic
